### ISSUE TITLE
[Request] Log tags that have been added manually

### ISSUE THREAD

**Issue by pastuh** [starik222/BooruDatasetTagManager#44]
So that you don't have to constantly check whether certain tags have already been added, it would be good to register them. 

- A simple popup with lists where tags can be selected (if needed)

**Comment by jojje** [comment]
I don't understand this one. If a tag has been added to any image, it will show up in the "All tags" list. If you've not assigned it to any image, it won't. 
If you try to assign a tag more than once it won't add the tag a second time (idempotent operation since tags are a set).

**Comment by jojje** [comment]
I don't understand this one. If a tag has been added to any image, it will show up in the "All tags" list. If you've not assigned it to any image, it won't.
If you try to assign a tag more than once it won't add the tag a second time (idempotent operation since tags are a set).

**Comment by pastuh** [comment]
It is listed, but the request is to add tags that were created manually to a separate list

**Comment by jojje** [comment]
You mean like your own little cheat-sheet / personalized set of tags that you only want to use for that tagging session? 
E.g. `blurry, plain background, <Class A>, <Class B>`, so that you don't have to then remember what tags you initially decided to constrain your tagging session to?

**Comment by pastuh** [comment]
Yes, like cheat-sheet. Indication that those tags are being used, which I entered manually.
It would be useful if I could edit/delete/use these tags whenever necessary (like in normal list)
